indico_toolkit/metrics/compare_ground_truth.py

```diff
@@ -0,0 +1,119 @@
+from indico_toolkit.association.association import sequences_overlap
+from indico_toolkit.types.extractions import Extractions
+from indico_toolkit.errors import ToolkitInputError
+
+
+class CompareGroundTruth:
+    """
+    Compare a set of ground truths against a set of model predictions on a per document basis.
+    """
+
+    def __init__(self, ground_truth: list, predictions: list):
```

Comment:
`from typing import List`
Then you can set the type hints to be `List[dict]` for each of these